frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetInstallSubmariner/InstallSubmarinerForm.tsx

```diff
@@ -1,18 +1,18 @@
 /* Copyright Contributors to the Open Cluster Management project */
 
-import { List, ListItem, PageSection } from '@patternfly/react-core'
-import { ExternalLinkAltIcon } from '@patternfly/react-icons'
-import { useContext, useEffect, useState } from 'react'
-import { useHistory } from 'react-router'
-import { Link } from 'react-router-dom'
-import { AcmDataFormPage } from '../../../../../../components/AcmDataForm'
-import { FormData, Section } from '../../../../../../components/AcmFormData'
-import { RbacButton } from '../../../../../../components/Rbac'
-import { Trans, useTranslation } from '../../../../../../lib/acm-i18next'
-import { DOC_LINKS } from '../../../../../../lib/doc-util'
-import { rbacCreate } from '../../../../../../lib/rbac-util'
-import { validateJSON, validateCloudsYaml, validateCidr } from '../../../../../../lib/validation'
-import { NavigationPath } from '../../../../../../NavigationPath'
+import {List, ListItem, PageSection} from '@patternfly/react-core'
+import {ExternalLinkAltIcon} from '@patternfly/react-icons'
+import {useContext, useEffect, useState} from 'react'
+import {useHistory} from 'react-router'
+import {Link} from 'react-router-dom'
+import {AcmDataFormPage} from '../../../../../../components/AcmDataForm'
+import {FormData, Section} from '../../../../../../components/AcmFormData'
+import {RbacButton} from '../../../../../../components/Rbac'
+import {Trans, useTranslation} from '../../../../../../lib/acm-i18next'
+import {DOC_LINKS} from '../../../../../../lib/doc-util'
+import {rbacCreate} from '../../../../../../lib/rbac-util'
+import {validateCidr, validateCloudsYaml, validateJSON} from '../../../../../../lib/validation'
```

Comment:
Do we need this change ?

Reply:
It was done by IDE during save, removed it.